app/Http/Resources/Experience.php

```diff
@@ -17,16 +17,45 @@ public function toArray($request)
     {
         $type = $this->experienceTypeName();
         $translations = [];
-        if ($type === 'experience_education') {
-            $translations = [
-                'education_status' => $this->education_status->getTranslations('name'),
-                'education_type' => $this->education_type->getTranslations('name'),
-            ];
-        } elseif ($type === 'experience_award') {
-            $translations = [
-                'award_recipient_type' => $this->award_recipient_type->getTranslations('name'),
-                'award_recognition_type' => $this->award_recognition_type->getTranslations('name'),
-            ];
+        $dates = [];
+        switch ($type) {
+            case 'experience_work':
+                $dates = [
+                    'start_date' => $this->when($this->start_date, $this->start_date->format('Y-m-d')),
+                    'end_date' => $this->when($this->end_date, $this->end_date->format('Y-m-d')),
```

Comment:
Oops! Turns out this causes an error when end_date is null. I guess `->when()` doesn't protect against null values? (To be fair, it seems to be intended for something different, ie relationships which may be loaded or not.)